README.rst

```diff
@@ -34,20 +34,21 @@ In order to use this library, you first need to go through the following steps:
 .. _Select or create a Cloud Platform project.: https://console.cloud.google.com/project
 .. _Enable billing for your project.: https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project
 .. _Enable the Google Cloud Storage API.:  https://cloud.google.com/storage
-.. _Setup Authentication.: https://googleapis.dev/python/google-api-core/latest/auth.html
+.. _Setup Authentication.: https://cloud.google.com/storage/docs/reference/libraries#setting_up_authentication
 
 Installation
 ~~~~~~~~~~~~
 
-Install this library in a `virtualenv`_ using pip. `virtualenv`_ is a tool to
-create isolated Python environments. The basic problem it addresses is one of
-dependencies and versions, and indirectly permissions.
+`Set up a Python development environment`_ and install this library in a `venv`.
+`venv`_ is a tool to create isolated Python environments. The basic problem it
+addresses is one of dependencies and versions, and indirectly permissions.
 
-With `virtualenv`_, it's possible to install this library without needing system
+With `venv`_, it's possible to install this library without needing system
 install permissions, and without clashing with the installed system
 dependencies.
 
-.. _`virtualenv`: https://virtualenv.pypa.io/en/latest/
+.. _Set up a Python development environment: https://cloud.google.com/python/docs/setup
```

Comment:
Can we add a line that says something along the lines of: "Make sure you're using Python 3.3 and later, which includes venv by default."

Reply:
Of course, done.